### PR TITLE
Qol improvements

### PR DESCRIPTION
**Pull request by agwosdz**
Changes include: 

- All True/False options are now "NOARG" eliminating the need to write True/False and only add the option (e.g. --keep_cache)

- Selected options are listed in STDOUT
{8BE4D0B6-1730-49DA-ACDE-044320D4DA47}

- Option to select Demucs model in parameters (experimental), as well as error detection for model name  
{A702F725-3944-4006-8348-86E0773CE765}

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,28 +103,32 @@ _Not all options working now!_
...
     [transcription]
     # Default is whisper
-    --whisper               Multilingual model > tiny|base|small|medium|large-v1|large-v2  >> ((default) is large-v2)
+    --whisper               Multilingual model > tiny|base|small|medium|large-v1|large-v2|large-v3  >> ((default) is large-v2)
                             English-only model > tiny.en|base.en|small.en|medium.en
     --whisper_align_model   Use other languages model for Whisper provided from huggingface.co
     --language              Override the language detected by whisper, does not affect transcription but steps after transcription
     --whisper_batch_size    Reduce if low on GPU mem >> ((default) is 16)
     --whisper_compute_type  Change to "int8" if low on GPU mem (may reduce accuracy) >> ((default) is "float16" for cuda devices, "int8" for cpu)
+    --keep_numbers          Numbers will be transcribed as numerics instead of as words >> ((default) is False)
```
Comment (rakuri255) [inline README.md:118]: so then remove `((default) is False)` ?

File: README.md [hunk 2]
```diff
@@ -103,28 +103,32 @@ _Not all options working now!_
...
     
     [extra]
-    --hyphenation           Use automatic hyphenation > True|False >> ((default) is True)
-    --disable_separation    Disable track separation > True|False >> ((default) is False)
-    --disable_karaoke       True|False >> ((default) is False)
-    --ignore_audio          True|False >> ((default) is False)
-    --create_audio_chunks   True|False >> ((default) is False)
-    --keep_cache            Keep cache folder after generation > True|False >> ((default) is False)
-    --plot                  Create a pitch plot > True|False >> ((default) is False)
+    --disable_hyphenation   Disable word hyphenation. Hyphenation is enabled by default.
```
Comment (rakuri255) [inline README.md:126]: We need also an update in the Readme in the cli sections in Hyphenate and Seperation

**Review by rakuri255 — CHANGES_REQUESTED** [review]
LGTM. I like this changes!

File: README.md (hunk 2 above)
Comment (agwosdz) [reply README.md:126]: Will add that

File: README.md (hunk 1 above)
Comment (agwosdz) [reply README.md:118]: Yeah, Sounds better that way

**Review by agwosdz — COMMENTED** [review]
All addressed

**Review by rakuri255 — APPROVED** [review]

**Pull request merged**